R/create_dcut.R

```diff
@@ -1,17 +1,24 @@
-#' Create DCUT dataset
+#' Create Datacut Dataset (DCUT)
 #'
-#' Use to create a datacut dataset containing the variables `USUBJID`, `DCUTDTC`, `DCUTDTM` and
-#' `DCUTDESC`.
+#' @description After filtering the input DS dataset (based on the given filter condition), any
+#' records where the SDTMv date variable is before the datacut date (after imputations)
+#' will be returned in the output datacut dataset (DCUT). Note that `ds_date_var` and
+#' `cut_date` inputs must be in ISO 860 format (YYYY-MM-DDThh:mm:ss) and will be imputed
+#' using the `impute_sdtm()` and `impute_dcutdtc()` functions.
```

Comment:
```suggestion
#' @description After filtering the input DS dataset (based on the given filter condition), any
#' records where the SDTMv date/time variable is on or before the datacut date/time (after imputations)
#' will be returned in the output datacut dataset (DCUT). Note that `ds_date_var` and
#' `cut_date` inputs must be in ISO 860 format (YYYY-MM-DDThh:mm:ss) and will be imputed
#' using the `impute_sdtm()` and `impute_dcutdtc()` functions.
```

Reply:
Hello @alanaharris22 - thanks for your updates. I have just suggested a few small cosmetic updates to the description above. If you agree with these changes, feel free to commit these suggestions and close this pull request.